index.html

```diff
@@ -3,22 +3,149 @@
+
+        <div class="teammate-info hidden">
+          <img
+            src="/images/febby-large.png"
+            alt="Febby's avatar in larger size."
+            class="expanded-avatar"
+          >
+          <p class="name">Febby</p>
+          <p class="secondary-text location">London, England</p>
+          <p class="secondary-text time-zone-reference">5:30pm (BST)</p>
+        </div>
+      </div>
+
+      <div class="teammate-avatar">
+        <img src="/images/augustin-small.png" alt="Augustin's avatar, it's a tiger.">
+
+        <div class="teammate-info hidden">
+          <img
+            src="/images/augustin-large.png"
+            alt="[teammate] avatar expanded"
+            class="expanded-avatar"
+          >
+          <p class="name">Augustin</p>
+          <p class="secondary-text location">Kathmandu, Nepal</p>
+          <p class="secondary-text time-zone-reference">10:15pm (NPT)</p>
+        </div>
+      </div>
+
+      <div class="teammate-avatar">
+        <img
+        src="/images/deercember-small.png" alt="Deercember's avatar, it's a deer.">
+
+        <div class="teammate-info hidden">
+          <img
+            src="/images/deercember-large.png" 
+            alt="Deercember's avatar in larger size."
+            class="expanded-avatar"
+          >
+          <p class="name">Deercember</p>
+          <p class="secondary-text location">Toronto, Canada</p>
+          <p class="secondary-text time-zone-reference">10:30pm (EDT)</p>
+        </div>
+      </div>
+
+      <div class="teammate-avatar hiring">
+        <p id="you">1+</p>
+        <div class="teammate-info hidden">
+          <p class="expanded-avatar">&#128075;</p>
+          <p class="name">You!</p>
+          <p class="secondary-text location">Anywhere</p>
+          <p class="secondary-text">We're 100% Remote</p>
+        </div>
+      </div>
+    </div>
 
-  <!-- Por aquí empieza el código. Con ese espacio alrededor el código es más legible y el doc más fácil de navegar. -->
+    <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
+  </nav>
 
+  <div id="hiring">
+    <div id="callAction">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <button> See the roles </button>
+    </div>
+      <div id="imageHiring">
+      <img src="images/joinbox-flying-shoe.png" alt="shoe">
+      <img src="images/joinbox-pencil-small.png" alt="pencil">
+      <img src="images/joinbox-abstract-green-box.png" alt="green">
+      <img src="images/joinbox-abstract-blue-ball.png" alt="blue">
+      <img src="images/joinbox-flying-febby.png" alt="">
+    </div>
+    <div id="form">
+      <h3> Join for early acess </h3>
+      <p> We're gradually opening Ready—get early access to updates and be a part of building the next generation of calendars.</p>
+      <form action="">
```

Comment:
Where's the placeholder that's in the original website's input field? Why did you leave it out? If you took it out because there was some problem, wouldn't you have used a label besides the input field instead?

Try to use the form yourself. Can you tell what you're suppossed to write in the input field without knowing your code? If you can't, how would your users do it?